tests_metricflow/snapshots/test_cumulative_metric_rendering.py/SqlQueryPlan/BigQuery/test_derived_cumulative_metric_with_non_default_grains__plan0_optimized.sql

```diff
@@ -11,19 +11,14 @@ FROM (
     -- Window Function for Metric Re-aggregation
     SELECT
       metric_time__week
-      , FIRST_VALUE(t2mr) OVER (
-        PARTITION BY metric_time__week
-        ORDER BY metric_time__day
-        ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING
-      ) AS t2mr
+      , AVG(t2mr) OVER (PARTITION BY metric_time__week) AS t2mr
     FROM (
       -- Join Self Over Time Range
       -- Pass Only Elements: ['txn_revenue', 'metric_time__week', 'metric_time__day']
       -- Aggregate Measures
       -- Compute Metrics via Expressions
       SELECT
-        subq_12.metric_time__day AS metric_time__day
```

Comment:
Wait, this doesn't look right. We're now doing the initial aggregation against WEEK instead of DAY, and then taking the average of those grouped by week, which is effectively a no-op. Is that expected?

Reply:
Shoot, you're right. I wonder if the column pruner got to it - investigating 🧐

Reply:
Definitely seems like an optimizer issue since the non-optimized SQL looks right.

Reply:
@tlento fixed this bug in the latest 2 commits.